docs/src/index.md

```diff
@@ -42,8 +42,14 @@ julia> BandedMatrix(Ones(5,5), (1,2))
            1.0  1.0  1.0
                 1.0  1.0
 ```
-
-
+To create a banded matrix of a given size with constant bands (such as the classical finite difference approximation of the two-dimensional Laplacian on the unit square [0,1]×[0,1]), you can use the following:
+```julia
+n = 128
+h = 1/n
+A = BandedMatrix{Float64}(undef, (n*n,n*n), (1,1))
```

Comment:
This should be an `(n,n)` not `(n*n,n*n)`.

Reply:
Note that -- differently from your example -- I am constructing the *2D* Laplacian (note the `-4.0` on the diagonal) which has size n^2 for an n-by-n grid (with mesh size 1/n).

But if you prefer, I can switch back to the 1D Laplacian. (I just thought the 2D case is more often used.)

Reply:
The 2D Laplacian is not a tridiagonal matrix: its block tridiagonal with tridiagonal blocks.

It is more naturally represented as a BandedBlockBandedMatrix from BlockBandedMatrices.jl